pkg/pot/lexer_test.go

```diff
@@ -0,0 +1,124 @@
+package pot
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLex(t *testing.T) {
```

Comment:
Add a test case for invalid pot as well. E.g.`msgstr"oops"`. Is it invalid by POT spec?